app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ActiveRecord::Base
   has_many :wikis
-  has_many :collaborators, through: :wikis
+  has_many :collaborating_wikis, through: :collaborators, foreign_key: "wiki_id", class_name: "Wiki"
+  
```

Comment:
@bjblock - I ended up researching the way to do this to avoid conflicts with the `wiki_id` and this is what I came up with. Does this look okay to you?